modules/packages/ExternalArray.chpl

```diff
@@ -302,8 +302,29 @@ module ExternalArray {
     var arr = new unmanaged ExternArr(eltType,
                                       dom,
                                       value,
-                                      false);
+                                      _owned=false);
     dom.add_arr(arr, locking = false);
     return _newArray(arr);
   }
+
+  proc makeReturnableArr(arr: []) {
```

Comment:
Maybe rename this function to refer to extern arrays, like convertToExternalArray?